### PR TITLE
[BULK] Validation: Description, tables and link fixes

### PR DESCRIPTION
**Pull request by SueADarby**
## Category

- [x] Content fix
- [] New article

## Related issues

No issue part of validation

## What's in this Pull Request?
 This PR fixes 32 issues for 11 files.

### CODE REVIEW

**Comment by VesaJuvonen** [comment]
Docs Build status updates of commit _[33ab6db](https://github.com/SueADarby/sp-dev-docs/commits/33ab6db101b82329383e6f2f8fb323686d77850f)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/schema/activationdependency-element-featureactivationdependencydefinitions-complextypea.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/activationdependency-element-featureactivationdependencydefinitions-complextypea.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/activationdependency-element-featureactivationdependencydefinitions-complextypea?branch=pr-en-us-8186) | [Details](#user-content-8384a5436251a71a782a452898ed6492-af3cebd1)
[docs/schema/activationdependency-element-solution.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/activationdependency-element-solution.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/activationdependency-element-solution?branch=pr-en-us-8186) | [Details](#user-content-9bb3a341a44cd73e9067ecc917830af2-af3cebd1)
[docs/schema/active-element-source-complextypesps15xsdsearchset1.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/active-element-source-complextypesps15xsdsearchset1.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/active-element-source-complextypesps15xsdsearchset1?branch=pr-en-us-8186) | [Details](#user-content-4677d5bd81b1710945d86e0fe3fb7589-af3cebd1)
[docs/schema/activitybody-element-parameter-elementcoercionparameterstype-complextypeaction4.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/activitybody-element-parameter-elementcoercionparameterstype-complextypeaction4.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/activitybody-element-parameter-elementcoercionparameterstype-complextypeaction4?branch=pr-en-us-8186) | [Details](#user-content-e5a54705e3f487352e3cd59d6b0b166e-af3cebd1)
[docs/schema/activitysource-element-evaluation-elementdatasourcetype-complextypeaction4.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/activitysource-element-evaluation-elementdatasourcetype-complextypeaction4.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/activitysource-element-evaluation-elementdatasourcetype-complextypeaction4?branch=pr-en-us-8186) | [Details](#user-content-9c69e0950f6b057109d1e4a0dc1fd8b6-af3cebd1)
[docs/schema/activitysource-element-workflowactions.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/activitysource-element-workflowactions.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/activitysource-element-workflowactions?branch=pr-en-us-8186) | [Details](#user-content-ffcf065415ae76bb8f0f33708a52044e-af3cebd1)
[docs/schema/activitysource-element-workflowinfo.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/activitysource-element-workflowinfo.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/activitysource-element-workflowinfo?branch=pr-en-us-8186) | [Details](#user-content-65d2f1bd60345fbb9fc38a0cf6d02799-af3cebd1)
[docs/schema/addcontenttypefield-element-feature.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/addcontenttypefield-element-feature.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/addcontenttypefield-element-feature?branch=pr-en-us-8186) | [Details](#user-content-4948ce49b6838fc920c604c1d42de13f-af3cebd1)
[docs/schema/aggregations-element-deployment-manifestspview.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/aggregations-element-deployment-manifestspview.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/aggregations-element-deployment-manifestspview?branch=pr-en-us-8186) | [Details](#user-content-e8a33bea6227fc3b4cc292e66abcc667-af3cebd1)
[docs/schema/aggregations-element-deploymentmanifestspwebpart.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/aggregations-element-deploymentmanifestspwebpart.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/aggregations-element-deploymentmanifestspwebpart?branch=pr-en-us-8186) | [Details](#user-content-20fe63d286af6af15d5d1568a35561b8-af3cebd1)
[docs/schema/alertchangetype-element-sps15xsdsearchset5.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/alertchangetype-element-sps15xsdsearchset5.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/alertchangetype-element-sps15xsdsearchset5?branch=pr-en-us-8186) | [Details](#user-content-39160b8b8cf14ecef49570f4f93d8198-af3cebd1)
[docs/schema/aliases-element-managedpropertyinfo-complextypesps15xsdsearchset2.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/aliases-element-managedpropertyinfo-complextypesps15xsdsearchset2.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/aliases-element-managedpropertyinfo-complextypesps15xsdsearchset2?branch=pr-en-us-8186) | [Details](#user-content-42d14d40e927df07e5f4f3744a3903fd-af3cebd1)
[docs/schema/aliases-element-searchschemaconfigurationsettings-complextypesps15xsdsearchset3.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/aliases-element-searchschemaconfigurationsettings-complextypesps15xsdsearchset3.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/aliases-element-searchschemaconfigurationsettings-complextypesps15xsdsearchset3?branch=pr-en-us-8186) | [Details](#user-content-ab09a18c9ed98727f519fbfae4a1f32f-af3cebd1)
[docs/schema/aliasesoverridden-element-overrideinfo-complextypesps15xsdsearchset2.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/aliasesoverridden-element-overrideinfo-complextypesps15xsdsearchset2.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/aliasesoverridden-element-overrideinfo-complextypesps15xsdsearchset2?branch=pr-en-us-8186) | [Details](#user-content-01ae09c095fb109e710f34dbd8569fe5-af3cebd1)
[docs/schema/aliasinfo-complextype-sps15xsdsearchset2.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/aliasinfo-complextype-sps15xsdsearchset2.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/aliasinfo-complextype-sps15xsdsearchset2?branch=pr-en-us-8186) | [Details](#user-content-6b0b0caf14f3031bd8103fc3ff166f6f-af3cebd1)
[docs/schema/aliasinfo-element-sps15xsdsearchset2.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/aliasinfo-element-sps15xsdsearchset2.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/aliasinfo-element-sps15xsdsearchset2?branch=pr-en-us-8186) | [Details](#user-content-3ab8244fa2776d222d569cb0ad83d35d-af3cebd1)
[docs/schema/aliasinfocollection-complextype-sps15xsdsearchset2.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/aliasinfocollection-complextype-sps15xsdsearchset2.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/aliasinfocollection-complextype-sps15xsdsearchset2?branch=pr-en-us-8186) | [Details](#user-content-d0cb9fc0c95dd6c1b249db8e8591e4eb-af3cebd1)
[docs/schema/aliasinfocollection-element-sps15xsdsearchset2.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/aliasinfocollection-element-sps15xsdsearchset2.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/aliasinfocollection-element-sps15xsdsearchset2?branch=pr-en-us-8186) | [Details](#user-content-bb724c9905aa5dc84fb8d0d0ed08d1b7-af3cebd1)

<a id="8384a5436251a71a782a452898ed6492-af3cebd1"></a>
### [docs/schema/activationdependency-element-featureactivationdependencydefinitions-complextypea.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/activationdependency-element-featureactivationdependencydefinitions-complextypea.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="9bb3a341a44cd73e9067ecc917830af2-af3cebd1"></a>
### [docs/schema/activationdependency-element-solution.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/activationdependency-element-solution.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="4677d5bd81b1710945d86e0fe3fb7589-af3cebd1"></a>
### [docs/schema/active-element-source-complextypesps15xsdsearchset1.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/active-element-source-complextypesps15xsdsearchset1.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="e5a54705e3f487352e3cd59d6b0b166e-af3cebd1"></a>
### [docs/schema/activitybody-element-parameter-elementcoercionparameterstype-complextypeaction4.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/activitybody-element-parameter-elementcoercionparameterstype-complextypeaction4.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="9c69e0950f6b057109d1e4a0dc1fd8b6-af3cebd1"></a>
### [docs/schema/activitysource-element-evaluation-elementdatasourcetype-complextypeaction4.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/activitysource-element-evaluation-elementdatasourcetype-complextypeaction4.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="ffcf065415ae76bb8f0f33708a52044e-af3cebd1"></a>
### [docs/schema/activitysource-element-workflowactions.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/activitysource-element-workflowactions.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="65d2f1bd60345fbb9fc38a0cf6d02799-af3cebd1"></a>
### [docs/schema/activitysource-element-workflowinfo.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/activitysource-element-workflowinfo.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="4948ce49b6838fc920c604c1d42de13f-af3cebd1"></a>
### [docs/schema/addcontenttypefield-element-feature.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/addcontenttypefield-element-feature.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="e8a33bea6227fc3b4cc292e66abcc667-af3cebd1"></a>
### [docs/schema/aggregations-element-deployment-manifestspview.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/aggregations-element-deployment-manifestspview.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="20fe63d286af6af15d5d1568a35561b8-af3cebd1"></a>
### [docs/schema/aggregations-element-deploymentmanifestspwebpart.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/aggregations-element-deploymentmanifestspwebpart.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="39160b8b8cf14ecef49570f4f93d8198-af3cebd1"></a>
### [docs/schema/alertchangetype-element-sps15xsdsearchset5.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/alertchangetype-element-sps15xsdsearchset5.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="42d14d40e927df07e5f4f3744a3903fd-af3cebd1"></a>
### [docs/schema/aliases-element-managedpropertyinfo-complextypesps15xsdsearchset2.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/aliases-element-managedpropertyinfo-complextypesps15xsdsearchset2.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="ab09a18c9ed98727f519fbfae4a1f32f-af3cebd1"></a>
### [docs/schema/aliases-element-searchschemaconfigurationsettings-complextypesps15xsdsearchset3.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/aliases-element-searchschemaconfigurationsettings-complextypesps15xsdsearchset3.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="01ae09c095fb109e710f34dbd8569fe5-af3cebd1"></a>
### [docs/schema/aliasesoverridden-element-overrideinfo-complextypesps15xsdsearchset2.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/aliasesoverridden-element-overrideinfo-complextypesps15xsdsearchset2.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="6b0b0caf14f3031bd8103fc3ff166f6f-af3cebd1"></a>
### [docs/schema/aliasinfo-complextype-sps15xsdsearchset2.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/aliasinfo-complextype-sps15xsdsearchset2.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="3ab8244fa2776d222d569cb0ad83d35d-af3cebd1"></a>
### [docs/schema/aliasinfo-element-sps15xsdsearchset2.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/aliasinfo-element-sps15xsdsearchset2.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="d0cb9fc0c95dd6c1b249db8e8591e4eb-af3cebd1"></a>
### [docs/schema/aliasinfocollection-complextype-sps15xsdsearchset2.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/aliasinfocollection-complextype-sps15xsdsearchset2.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="bb724c9905aa5dc84fb8d0d0ed08d1b7-af3cebd1"></a>
### [docs/schema/aliasinfocollection-element-sps15xsdsearchset2.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/aliasinfocollection-element-sps15xsdsearchset2.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/8c4da01d-e430-7fe7-0bea-2c4100cce9e8/202206141747356993-8186/BuildReport?accessString=9e2990124f65a13b6e476ac4bcc190947756fe3ea03ec5b3d33d521f6fc71948).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=main) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/8c4da01d-e430-7fe7-0bea-2c4100cce9e8?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=main&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/schema/activationdependency-element-solution.md [hunk 1]
```diff
@@ -36,10 +36,10 @@ The following sections describe attributes, child elements, and parent elements.
 
 |**Attribute**|**Description**|
 |:-----|:-----|
-|**SolutionId** <br/> |Required **string**. Specifies the solution ID of the dependency.  <br/> |
-|**SolutionName** <br/> |Optional **string**. Specifies the name of the solution dependency.  <br/> |
-|**SolutionTitle** <br/> |Optional **string**. Specifies the title of the solution dependency.  <br/> |
-|**SolutionUrl** <br/> |Optional **string**. Specifies the URL of the solution dependency.  <br/> |
+|**SolutionId**  |Required **string**. Specifies the solution ID of the dependency.   |
```
Comment (RedZephyr13) [inline docs/schema/activationdependency-element-solution.md:39]: No need to fix these tables that are out of scope.

**Review by andrewconnell — APPROVED** [review]
